Chefs/Views/LiveCookingPage.xaml

```diff
@@ -77,11 +78,14 @@
 						</muxc:ItemsRepeater.ItemTemplate>
 					</muxc:ItemsRepeater>
 				</utu:AutoLayout>
-				<TextBlock Foreground="{ThemeResource OnSurfaceBrush}"
-						   Style="{StaticResource TitleMedium}"
+				<ScrollViewer VerticalScrollBarVisibility="Auto"
```

Comment:
It seems like the FlipView is highjacking the scroll so even if my text is cut off and I try and scroll while hovered over this, the FlipView is what gets scrolled and not the step text:

<img width="421" alt="image" src="https://github.com/user-attachments/assets/ba066d09-06bf-4e1d-8e27-77ff8c22a1e1" />

So I think there may not be a good way to introduce a ScrollViewer here :/